lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
No, one half of the contacts are deleted via `delete_contacts_of_me` and the second half is deleted with deleting aspects in `delete_standard_user_associations`. And when I wanted to delete it a third time in `disconnect_contacts` it fails with:
```
NoMethodError: undefined method `disconnect' for nil:NilClass
./app/models/aspect_membership.rb:14:in `block in <class:AspectMembership>'
...
./lib/account_deleter.rb:73:in `disconnect_contacts'
./lib/account_deleter.rb:37:in `block in perform!'
...
```

When I do `user.contacts.reload.destroy_all` in `disconnect_contacts` it works, but there are no contacts left.